src/opm/parser/eclipse/share/keywords/keyword_list.cmake

```diff
@@ -9,6 +9,25 @@
 # you are therefore encouraged to go to the opm-simulators repository and update
 # the file: opm/autodiff/MissingFeatures.cpp
 
+#Some keywords are found to be of 'special' structure:
+#W             WHAT
+#ECONT         Terminates with two /
+#COALNUM       matrix structure
+#COMPDATM      alias for COMPDATL
+#COMPINJK      undef default
+#COMPRP        strange defaults
+#COMPRPL       strange defaults
+#COMPVE        default: negative
+#COMPVEL       default: negative
+#CSKIN         strange default
+#DEPTHTAB      different unit in table
```

Comment:
DEPTHTAB looks tricky - I would just go for `"SIZE_TYPE":"ALL"` and `"value_type":"STRING"` - i.e. completely ignore all unit conversion.